integrations/nvim/queries/tlaplus/highlights.scm

```diff
@@ -136,48 +136,55 @@
 (escape_char) @string.escape
 (string_set) @type
 
-; Namespaces and includes
-(extends (identifier_ref) @include)
-(module name: (_) @namespace)
+; Namespaces
+(extends (identifier_ref) @namespace)
+(instance (identifier_ref) @namespace)
+(module name: (identifier) @namespace)
 (pcal_algorithm name: (identifier) @namespace)
 
-; Operators and functions
-(bound_infix_op symbol: (_) @function.builtin)
-(bound_nonfix_op (infix_op_symbol) @operator)
-(bound_nonfix_op (postfix_op_symbol) @operator)
-(bound_nonfix_op (prefix_op_symbol) @operator)
-(bound_postfix_op symbol: (_) @function.builtin)
-(bound_prefix_op symbol: (_) @function.builtin)
+; Operators, functions, and macros
+(bound_infix_op symbol: (_) @operator)
+(bound_nonfix_op symbol: (_) @operator)
+(bound_postfix_op symbol: (_) @operator)
+(bound_prefix_op symbol: (_) @operator)
+((prefix_op_symbol) @operator)
+((infix_op_symbol) @operator)
+((postfix_op_symbol) @operator)
 (function_definition name: (identifier) @function)
-(module_definition name: (identifier) @function)
-(operator_definition name: (_) @operator)
+(module_definition name: (_) @include)
+(operator_definition name: (_) @function.macro)
 (pcal_macro_decl name: (identifier) @function.macro)
 (pcal_macro_call name: (identifier) @function.macro)
 (pcal_proc_decl name: (identifier) @function.macro)
 (pcal_process name: (identifier) @function)
-(recursive_declaration (identifier) @operator)
-(recursive_declaration (operator_declaration name: (_) @operator))
+(recursive_declaration (identifier) @function.macro)
+(recursive_declaration (operator_declaration name: (_) @function.macro))
 
 ; Constants and variables
-(constant_declaration (identifier) @constant.builtin)
-(constant_declaration (operator_declaration name: (_) @constant.builtin))
-(pcal_var_decl (identifier) @variable.builtin)
+(constant_declaration (identifier) @constant)
+(constant_declaration (operator_declaration name: (_) @constant))
+(pcal_var_decl (identifier) @variable)
 (pcal_with (identifier) @parameter)
 ((".") . (identifier) @attribute)
 (record_literal (identifier) @attribute)
 (set_of_records (identifier) @attribute)
-(variable_declaration (identifier) @variable.builtin)
+(variable_declaration (identifier) @variable)
 
 ; Parameters
-(quantifier_bound (identifier) @parameter)
-(quantifier_bound (tuple_of_identifiers (identifier) @parameter))
+(choose (identifier) @parameter)
```

Comment:
Shouldn't these changes also apply to nvim highlight queries ?

Reply:
You mean the non-nvim highlight queries? The ones in integrations/nvim/queries are for nvim. Yeah I've just been focusing on the nvim queries then will transfer the changes over to the non-nvim queries.